public/v4/apps/plausible.yml

```diff
@@ -72,27 +73,31 @@ caproverOneClickApp:
           id: $$cap_RELAY_PASSWORD
           label: RELAY_PASSWORD
           validRegex: /^([^\s^\/])+$/
-        - description: The version of Plausible docker image (from https://hub.docker.com/r/plausible/analytics/tags e.g. v2.0)
+        - description: The version of Plausible docker image (from https://github.com/plausible/analytics/pkgs/container/community-edition e.g. v2.1.1)
           id: $$cap_PLAUSIBLE_VERSION
           label: PLAUSIBLE_VERSION
-          defaultValue: v2.0
+          defaultValue: v2.1.1
           validRegex: /^([^\s^\/])+$/
-        - description: Clickhouse version (from https://hub.docker.com/r/clickhouse/clickhouse-server/tags, e.g. 23.3.7.5-alpine)
+        - description: Clickhouse version (from https://hub.docker.com/r/clickhouse/clickhouse-server/tags, e.g. 24.3.3.102-alpine)
           id: $$cap_CLICKHOUSE_VERSION
           label: CLICKHOUSE_VERSION
-          defaultValue: 23.3.7.5-alpine
+          defaultValue: 24.3.3.102-alpine
           validRegex: /^([^\s^\/])+$/
-        - description: Postgres version (from https://hub.docker.com/_/postgres?tab=tags, supported versions are 12, 13, and 14)
+        - description: Postgres version (from https://hub.docker.com/_/postgres?tab=tags, Plausible v2.1.1 was tested against PostgreSQL versions 15 and 16)
           id: $$cap_POSTGRES_VERSION
           label: POSTGRES_VERSION
-          defaultValue: 14-alpine
+          defaultValue: 16-alpine
           validRegex: /^([^\s^\/])+$/
+        - description: This new key is used to encrypt TOTP secrets at rest. Generate by running $ openssl rand -base64 32
+          id: $$cap_TOTP_VAULT_KEY
+          label: TOTP_VAULT_KEY
+          defaultValue: aihU7k+GSBKbcVFuX9VEPyUhuwlKEomqA94/SQQ0NR4=
```

Comment:
I didn't notice this hardcoded value as a default value. It poses a security risk. I'll remove this default value.